storage/app/src/main/java/com/google/firebase/referencecode/storage/KotlinStorageActivity.kt

```diff
@@ -0,0 +1,405 @@
+package com.google.firebase.referencecode.storage
+
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.net.Uri
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.ImageView
+import com.google.android.gms.tasks.Continuation
+import com.google.android.gms.tasks.Task
+import com.google.firebase.referencecode.storage.interfaces.StorageActivityInterface
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageMetadata
+import com.google.firebase.storage.StorageReference
+import com.google.firebase.storage.UploadTask
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.FileInputStream
+
+class KotlinStorageActivity : AppCompatActivity(), StorageActivityInterface {
+    // [START storage_field_declaration]
```

Comment:
Maybe we should make the field a `lateiniit var` and then initialize it in `onCreate` rather than have an empty snippet here?

Reply:
Sure, will do that.